### PR TITLE
COG-1096 Accout Status Changed Event

### PR DESCRIPTION
**Pull request by AlexanderSerbin**
### JIRA Issue Link
https://osi-cwds.atlassian.net/browse/COG-1096

### Technical Description
Added Account Status Changed Event

### Tests
- [x] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package gov.ca.cwds.idm.service.cognito.attribute;
+
+/**
+ * Created by Alexander Serbin on 1/21/2019
+ */
+public enum OtherUserAttribute implements UserAttribute {
+
+  ACCOUNT_STATUS("account_status");
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java]: Please don't use term  ACCOUNT_STATUS for things which are not account status

Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java]: What is the correct term?

Comment (dmytro-palczewski-osi) [reply src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java]: may be ENABLED STATUS?

File: src/main/java/gov/ca/cwds/idm/service/UserUpdateRequest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+/**
+ * Created by Alexander Serbin on 1/20/2019
+ */
+public class UserUpdateRequest {
+
+  private UserType existedCognitoUser;
+  private String userId;
+  private Map<UserAttribute, UserAttributeDiff> diffMap;
+  private User user;
```
Comment (dmytro-palczewski-osi) [inline src/main/java/gov/ca/cwds/idm/service/UserUpdateRequest.java]: should be renamed to existedUser

File: src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java (hunk 1 above)
Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/cognito/attribute/OtherUserAttribute.java]: Done

File: src/main/java/gov/ca/cwds/idm/service/UserUpdateRequest.java (hunk 2 above)
Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/UserUpdateRequest.java]: Done

**Pull request merged**